security.rst

```diff
@@ -7,6 +7,6 @@
 See the IPython documentation for more information on `notebook security`_,
 or `reporting security issues`_
 
-.. _notebook security: http://ipython.org/ipython-doc/2/notebook/security.html
-.. _reporting security issues: http://ipython.org/ipython-doc/2/notebook/security.html#reporting-security-issues
+.. _notebook security: https://ipython.org/ipython-doc/2/notebook/security.html
```

Comment:
This should probably point to https://jupyter-notebook.readthedocs.io/en/stable/security.html now, right?

Reply:
Yeah... old local changes.